### PR TITLE
Copy over filetype settings from vim-elixir

### PR DESCRIPTION
**Pull request by goto-engineering**
As discussed with @mhanberg, the maintainer

### CODE REVIEW

File: runtime/ftplugin/elixir.vim [hunk 1]
```diff
@@ -23,7 +23,41 @@ if exists('loaded_matchit') && !exists('b:match_words')
...
+let &l:path =
+      \ join([
+      \   'lib/**',
+      \   'src/**',
+      \   'test/**',
+      \   'deps/**/lib/**',
+      \   'deps/**/src/**',
+      \   &g:path
+      \ ], ',')
+setlocal includeexpr=elixir#util#get_filename(v:fname)
```
Comment (mhanberg) [inline runtime/ftplugin/elixir.vim]: This function doesn't exist.

Comment (goto-engineering) [reply runtime/ftplugin/elixir.vim]: Ah, it seems to refer to this:
https://github.com/elixir-editors/vim-elixir/blob/527e6fd8798638a79621e0b5c788b67b2b4b4dbc/autoload/elixir/util.vim

**Review by mhanberg — APPROVED** [review]

**Comment by brammool** [comment]
This plugin should set b:undo_ftplugin

**Comment by mhanberg** [comment]
@goto-engineering please add the following to the bottom of the file

```vim
let b:undo_ftplugin = 'setlocal sw< sts< et< isk< com< cms<'
```

**Comment by brammool** [comment]
I'll include it, thanks.

**Pull request closed without merging**